visualization/app/codeCharta/ui/codeMap/threeViewer/threeViewerService.ts

```diff
@@ -52,10 +123,26 @@ export class ThreeViewerService {
 		}
 	}
 
+	private render() {
+		if (this.enableFXAA) {
+			this.threeRendererService.composer.render()
+		} else {
+			this.threeRendererService.renderer.render(this.threeSceneService.scene, this.threeCameraService.camera)
+		}
+	}
+
+	dispose() {
+		// TODO needs to bind it
+		if (this.threeRendererService !==undefined && this.threeRendererService.composer!==undefined) {
```

Comment:
I think, prettier will do this for you:
```suggestion
		if (this.threeRendererService !== undefined && this.threeRendererService.composer !== undefined) {
```